lib/msal-common/src/cache/entities/AccountEntity.ts

```diff
@@ -143,6 +146,7 @@ export class AccountEntity {
 
         account.environment = env;
         account.realm = idToken.claims.tid;
+        account.oboAssertion = oboAssertion;
```

Comment:
Is this not needed for ADFS cases?

Reply:
Yes good point, I have added for ADFS2019 accounts as well.